cmd/controller/main.go

```diff
@@ -46,6 +47,7 @@ var ctors = []injection.ControllerConstructor{
 	gc.NewController,
 	nscert.NewController,
 	domainmapping.NewController,
+	certificate.NewController, // net-certmanager controller
```

Comment:
maybe just rename the package, as the name "net-certmanager" will vanish with this PR.

Reply:
Yeah good point, I was thinking the same as it is not readable.